backend/src/routes/tag.ts

```diff
@@ -39,6 +38,45 @@ tagRouter.get("/", async (c) => {
 	}
 });
 
+tagRouter.get("/recommended", async (c) => {
+	try {
+		const prisma = new PrismaClient({
+			datasourceUrl: c.env.DATABASE_URL,
+		}).$extends(withAccelerate());
+
```

Comment:
You can write this
```bash
const prisma = getDBInstance(c);
```
instead of 
```bash
const prisma = new PrismaClient({
			datasourceUrl: c.env.DATABASE_URL,
		}).$extends(withAccelerate());		
```

Short form to do it.

Reply:
Sure. Should I make this change in other routes too?

Reply:
Yes